workflows/sarek/main.nf

```diff
@@ -794,7 +794,7 @@ workflow SAREK {
 
         // POST VARIANTCALLING
         POST_VARIANTCALLING(BAM_VARIANT_CALLING_GERMLINE_ALL.out.vcf_all,
-                            params.concatenate_vcfs)
+             fasta, params.concatenate_vcfs)
```

Comment:
```suggestion
                fasta,
                params.concatenate_vcfs)
```

Reply:
little issue with spacing, so let's try that

Reply:
Easiest way to appease @maxulysse is to give him the space he needs 😁